src/UI-Component/Adminpanel/main.js

```diff
@@ -29,31 +34,44 @@ const MainPage = () => {
   return (
     <div className="mt-20">
       <div>
-        <FirstPane />
+        <AdminIcons />
       </div>
 
       <div className="flex flex-col sm:flex-row md:flex-row lg:flex-row justify-center items-center py-5 space-y-2 sm:space-x-2 sm:space-y-0">
         <Button name="adminHome" activeButton={activeButton} onClick={handleButtonClick} />
-        <Button name="verifyLoan" activeButton={activeButton} onClick={handleButtonClick}></Button>
-        <Button name="loanApplications" activeButton={activeButton} onClick={handleButtonClick}></Button>
-        <Button name="currentLoans" activeButton={activeButton} onClick={handleButtonClick}></Button>
-        <Button name="loanApproval" activeButton={activeButton} onClick={handleButtonClick}></Button>
-        <Button name="specialRequest" activeButton={activeButton} onClick={handleButtonClick}></Button>
-        <Button name="postTransactionForm" activeButton={activeButton} onClick={handleButtonClick}></Button>
+        <Button name="verifyLoan" activeButton={activeButton} onClick={handleButtonClick} />
+        <Button name="loanApplications" activeButton={activeButton} onClick={handleButtonClick} />
+        <Button name="currentLoans" activeButton={activeButton} onClick={handleButtonClick} />
+        <Button name="loanApproval" activeButton={activeButton} onClick={handleButtonClick} />
+        <Button name="specialRequest" activeButton={activeButton} onClick={handleButtonClick} />
+        <Button name="postTransactionForm" activeButton={activeButton} onClick={handleButtonClick} />
       </div>
 
       <div className="flex flex-col items-center">
-        {/* Render the active component based on the selected button */}
-        {activeButton === "verifyLoan" && <ViewVerified />}
-        {activeButton === "loanApplications" && <AllLoanApplications />}
-        {activeButton === "currentLoans" && <CurrentLoan />}
-        {activeButton === "adminHome" && <AdminHome />}
-        {activeButton === "loanApproval" && <LoanApprovalPage />}
-        {activeButton === "specialRequest" && <SpecialLoanRequest />}
-        {activeButton === "postTransactionForm" && <PostTransactionForm />}
-      </div>
+  {(() => {
+    switch (activeButton) {
```

Comment:
This looks sweet but this switch statement can be in an util (helper) file where you can import and use it here. Makes this component shorter